<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Presence</class>
 <widget class="QWidget" name="Presence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QRadioButton" name="m_pPresentRB">
     <property name="text">
      <string>Present</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="m_pAbsentRB">
     <property name="text">
      <string>Absent</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="m_pCustomMessageCK">
     <property name="text">
      <string>Custom</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="KTextEdit" name="m_pMessage">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QListView" name="m_pView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_pCustomMessageCK</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pMessage</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pCustomMessageCK</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pView</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pCustomMessageCK</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pPresentRB</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pCustomMessageCK</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pAbsentRB</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
